lib/ansible/modules/system/filesystem.py

```diff
@@ -22,7 +22,7 @@
 version_added: "1.2"
 options:
   fstype:
-    choices: [ btrfs, ext2, ext3, ext4, ext4dev, lvm, reiserfs, xfs, vfat ]
+    choices: [ btrfs, ext2, ext3, ext4, ext4dev, lvm, reiserfs, xfs, vfat, ocfs2 ]
     description:
     - Filesystem type to be created.
```

Comment:
@lchantre could you add a line in `description` mentioning something like `ocfs2 support was added in 2.6`. ?